homeassistant/components/rachio/binary_sensor.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, controller):
     @property
     def name(self) -> str:
         """Return the name of this sensor including the controller name."""
-        return f"{self._controller.name} online"
+        return f"{self._controller.name} status"
```

Comment:
Proposed change here. The name "online" to me indicates that the device is actually online, when it may not be. The question is if we change the name should we change the ID as well, which would probably be a breaking change?